js/src/modals/Verification/verification.js

```diff
@@ -20,12 +20,27 @@ import DoneIcon from 'material-ui/svg-icons/action/done-all';
 import CancelIcon from 'material-ui/svg-icons/content/clear';
 
 import { Button, IdentityIcon, Modal } from '~/ui';
+import RadioButtons from '~/ui/Form/RadioButtons';
+import { nullableProptype } from '~/util/proptypes';
+
+import styles from './verification.css';
+
+const methods = {
+  sms: {
+    label: 'SMS Verification', key: 0, value: 'sms',
+    description: (<p className={ styles.noSpacing }>It will be stored on the blockchain that you control a phone number (not <em>which</em>).</p>)
```

Comment:
Not a crisis for this PR since this one straddled the boundary between merges, but going forward when touching components change the text into `FormattedMessages`.